tests/fixed_point/test_fp_arithmetic_sugar.py

```diff
@@ -157,6 +66,35 @@ def test_add_fail(self):
         with self.assertRaises(TypeError):
             _ = int_value + fixed_point_value  # type: ignore
 
+    def test_add_nonfinite(self):
+        """Test rules for non-finite addition"""
+        # nan + anything is nan
+        assert (self.NAN + self.ZERO).is_nan() is True
```

Comment:
You don't need to explicitly test `is True`, the statement evaluates itself as true or false, which assert then passes or fails depending on the output. See rest of file for this as well.

Reply:
I know, I thought it was more clear to be explicit here for someone reading through the file. i.e. I liked this better than `assert not FALSE_STATEMENT` and `assert TRUE_STATEMENT`. I know it's atypical, I just thought it was easier to parse in such a test.